### ISSUE TITLE
Notion of ownership for Creations

### ISSUE THREAD

**Issue by sohkai** [bigchaindb/pycoalaip#6]
Ask Greg.

Context: What does ownership mean for Works and Manifestions? Who creates them (e.g. a COALA super user, or the actual requesting user)?

**Comment by sohkai** [comment]
Based on a discussion with @TimDaub and @gmcmullen today:

**TL;DR Diagram**, where everything in the **dotted** box is meaningful to own:

![Alt text](http://g.gravizo.com/g?
  digraph G {
    aize ="4,4";
    Work [shape=box];
    Work -> Manifestation;
    Manifestation -> Copyright;
    subgraph cluster_ownership {
        Copyright -> Right;
        graph[style=dotted];
    }
  }
)

**Results:**

* `Work` entities should only come into existence when the first `Manifestation` is registered for them
* Ownership of a `Work` or `Manifestation` entity is more or less meaningless in terms of rights
* A `Copyright` (subclass of `Right`) is created upon registration of a `Manifestation` and represents the `Manifestation`'s full copyright rights
   * Transferring a `Copyright` is the equivalent of transferring a `Manifestation`'s full copyrights to someone else
* `Right`s (thought of as licenses) are derived from the `Copyright` and are subsets of the full `Copyright`
     * `Right`s are creatable by the `Copyright` holder and transferrable to other `Parties`, barring contract restrictions
* **In the future**, `Right` can also be subclassed to provide more detailed rights
* **In the future**, derivative `Manifestation`s (of other `Manifestation`s) can be supported by linking them to their parent `Manifestation`s; these derivatives will have their own `Copyright`s

Based on this, the notion of ownership over a `Work` and a `Creation` is rather meaningless; it can either be registered to the original `Copyright` holder, to some arbitrary holding account (i.e. public key), or to no owner at all. Only the `Copyright` and further `Right`s are assets that are valuable to timestamp, own, and maintain provenance chains for. The `Work` and `Creation` can one day live in authenticated data stores, such as IPFS.
